diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/alloy.py ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/alloy.py
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/alloy.py	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/alloy.py	2023-01-01 23:55:34.000000000 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#! /usr/bin/python3 -sP
 #
 #  Copyright (c) 2013-2022, Intel Corporation
 #  All rights reserved.
diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/bitcode2cpp.py ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/bitcode2cpp.py
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/bitcode2cpp.py	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/bitcode2cpp.py	2023-01-01 23:55:34.000000000 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#! /usr/bin/python3 -sP
 
 import sys
 import re
diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/check_env.py ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/check_env.py
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/check_env.py	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/check_env.py	2023-01-01 23:55:34.000000000 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#! /usr/bin/python3 -sP
 #
 #  Copyright (c) 2013-2020, Intel Corporation
 #  All rights reserved.
diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/CMakeLists.txt ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/CMakeLists.txt
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/CMakeLists.txt	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/CMakeLists.txt	2023-01-07 14:06:56.078835878 -0800
@@ -36,8 +36,8 @@
 cmake_minimum_required(VERSION 3.13)
 
 if (UNIX)
-    set(CMAKE_C_COMPILER "clang")
-    set(CMAKE_CXX_COMPILER "clang++")
+    set(CMAKE_C_COMPILER "gcc")
+    set(CMAKE_CXX_COMPILER "g++")
 endif()
 
 set(PROJECT_NAME ispc)
@@ -289,10 +289,7 @@
     list(APPEND ISPC_TARGETS wasm-i32x4)
 endif()
 
-set(CLANG_LIBRARY_LIST clangFrontend clangDriver clangSerialization clangParse clangSema clangAnalysis clangAST clangBasic clangEdit clangLex)
-if (${LLVM_VERSION_NUMBER} VERSION_GREATER_EQUAL "15.0.0")
-    list(APPEND CLANG_LIBRARY_LIST clangSupport)
-endif()
+set(CLANG_LIBRARY_LIST clang-cpp)
 
 set(LLVM_COMPONENTS engine ipo bitreader bitwriter instrumentation linker option frontendopenmp)
 if (XE_ENABLED AND ${LLVM_VERSION_NUMBER} VERSION_GREATER_EQUAL "14.0.0")
@@ -513,7 +510,7 @@
 
 if (NOT MSVC)
     set_target_properties(${PROJECT_NAME} PROPERTIES CXX_EXTENSIONS OFF)
-    target_compile_options(${PROJECT_NAME} PRIVATE -Wno-c99-extensions -Wno-deprecated-register -fno-rtti)
+    target_compile_options(${PROJECT_NAME} PRIVATE  -fno-rtti)
     if (ISPC_USE_ASAN)
         target_compile_options(${PROJECT_NAME} PRIVATE -fsanitize=address)
     endif()
Only in ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original: CMakeLists.txt.rej
diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/common.py ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/common.py
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/common.py	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/common.py	2023-01-01 23:55:34.000000000 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#! /usr/bin/python3 -sP
 #
 #  Copyright (c) 2013-2022, Intel Corporation
 #  All rights reserved.
diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/perf.py ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/perf.py
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/perf.py	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/perf.py	2023-01-01 23:55:34.000000000 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#! /usr/bin/python3 -sP
 #
 #  Copyright (c) 2013-2020, Intel Corporation
 #  All rights reserved.
diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/run_tests.py ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/run_tests.py
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/run_tests.py	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/run_tests.py	2023-01-01 23:55:34.000000000 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#! /usr/bin/python3 -sP
 #
 #  Copyright (c) 2013-2022, Intel Corporation
 #  All rights reserved.
diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/stdlib2cpp.py ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/stdlib2cpp.py
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/stdlib2cpp.py	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/stdlib2cpp.py	2023-01-01 23:55:34.000000000 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#! /usr/bin/python3 -sP
 
 import sys
 
diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/utils/lit/lit/ProgressBar.py ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/utils/lit/lit/ProgressBar.py
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/utils/lit/lit/ProgressBar.py	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/utils/lit/lit/ProgressBar.py	2023-01-01 23:55:34.000000000 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#! /usr/bin/python3 -sP
 
 # Source: http://code.activestate.com/recipes/475116/, with
 # modifications by Daniel Dunbar.
diff -ru ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/utils/lit/lit.py ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/utils/lit/lit.py
--- ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e.original/utils/lit/lit.py	2023-01-01 23:55:34.000000000 -0800
+++ ispc-ec62d6cbef2fab4c49003c0206716d3d6248b59e/utils/lit/lit.py	2023-01-01 23:55:34.000000000 -0800
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#! /usr/bin/python3 -sP
 
 from lit.main import main
 
