diff -ru ispc-1.12.0.orig/src/llvmutil.cpp ispc-1.12.0/src/llvmutil.cpp
--- ispc-1.12.0.orig/src/llvmutil.cpp	2019-08-15 14:18:07.000000000 -0700
+++ ispc-1.12.0/src/llvmutil.cpp	2019-08-16 20:12:51.459626813 -0700
@@ -1041,11 +1042,11 @@
 
 #if 0
     fprintf(stderr, "all equal: ");
-    v->dump();
+    v->print(llvm::errs());
     fprintf(stderr, "\n");
     llvm::Instruction *inst = llvm::dyn_cast<llvm::Instruction>(v);
     if (inst) {
-        inst->getParent()->dump();
+        inst->getParent()->print(llvm::errs());
         fprintf(stderr, "\n");
         fprintf(stderr, "\n");
     }
@@ -1305,11 +1306,11 @@
 
 #if 0
     fprintf(stderr, "linear check: ");
-    v->dump();
+    v->print(llvm::errs());
     fprintf(stderr, "\n");
     llvm::Instruction *inst = llvm::dyn_cast<llvm::Instruction>(v);
     if (inst) {
-        inst->getParent()->dump();
+        inst->getParent()->print(llvm::errs());
         fprintf(stderr, "\n");
         fprintf(stderr, "\n");
     }
@@ -1349,7 +1350,7 @@
         return;
 
     fprintf(stderr, "  ");
-    v->dump();
+    v->print(llvm::errs());
     done.insert(v);
 
     if (inst == NULL)
diff -ru ispc-1.12.0.orig/src/opt.cpp ispc-1.12.0/src/opt.cpp
--- ispc-1.12.0.orig/src/opt.cpp	2019-08-15 14:18:07.000000000 -0700
+++ ispc-1.12.0/src/opt.cpp	2019-08-16 20:53:20.530268411 -0700
@@ -411,7 +411,7 @@
             llvm::TargetMachine *targetMachine = g->target->GetTargetMachine();
             const llvm::TargetData *td = targetMachine->getTargetData();
             llvm::Constant *c = llvm::ConstantFoldConstantExpression(ce, td);
-            c->dump();
+            c->print(llvm::errs());
             factor = c;
         }
         // else we should be able to handle it above...
@@ -515,7 +515,7 @@
 #ifndef ISPC_NO_DUMPS
     if (g->debugPrint) {
         printf("*** Code going into optimization ***\n");
-        module->dump();
+        module->print(llvm::errs(), nullptr);
     }
 #endif
     DebugPassManager optPM;
@@ -902,7 +902,7 @@
 #ifndef ISPC_NO_DUMPS
     if (g->debugPrint) {
         printf("\n*****\nFINAL OUTPUT\n*****\n");
-        module->dump();
+        module->print(llvm::errs(), nullptr);
     }
 #endif
 }
@@ -1869,7 +1869,7 @@
 static llvm::Value *
 lExtractUniforms(llvm::Value **vec, llvm::Instruction *insertBefore) {
     fprintf(stderr, " lextract: ");
-    (*vec)->dump();
+    (*vec)->print(llvm::errs());
     fprintf(stderr, "\n");
 
     if (llvm::isa<llvm::ConstantVector>(*vec) ||
